spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/pubsub/GcpPubSubAutoConfiguration.java

```diff
@@ -118,20 +121,27 @@ public ExecutorProvider subscriberExecutorProvider() {
 				this.gcpPubSubProperties.getSubscriber().getExecutorThreads()));
 	}
 
+	@Bean
+	@ConditionalOnBean(type = "com.fasterxml.jackson.databind.ObjectMapper")
```

Comment:
Looks like my comment on some of your commit has lost: https://github.com/spring-cloud/spring-cloud-gcp/pull/759#discussion_r196527435